ocaml/typing/includemod.ml

```diff
@@ -613,57 +654,59 @@ and functor_param ~in_eq ~loc env ~mark subst param1 param2 =
       let env, subst =
         match name1, name2 with
         | Some id1, Some id2 ->
-            Env.add_module id1 Mp_present arg2' env,
+            Env.add_module_lazy ~update_summary:true id1 Mp_present arg2' env,
```

Comment:
I pushed a commit to change this to `true`. Which places do you mean in your last sentence, though?

Reply:
The environment can appear in `Tcoerce_alias` or in the `pc_env` field under a `Tcoerce_primitive`. The first of those could be removed by replacing the `Env.t` and `Path.t` by a single `Env.address` via `Env.find_module_address` since that is all they are ever used for. The second one is much harder to remove but I think it would be fine to say that the contents of `pc_env` are not guaranteed to have summaries.

Reply:
Is this worth doing as part of this PR?

Reply:
Probably not. If we could get rid of both occurrences then I'd say it was worth it, but since we can only get rid of one we might as well leave it as it is for now.